client/app/location/location.js

```diff
@@ -0,0 +1,64 @@
+angular.module('dinnerDaddy.location', [])
+
+.controller('locationController', function ($scope, $location, $cookies, Location) {
+
+  $scope.username = $cookies.get('name');
+
+  $scope.verify = function (username) {
```

Comment:
Can we change this to local scope (just use var instead of $scope). We only need to put things on $scope if they need to be accessed in the HTML.